usmqe_tests/api/gluster/test_volume.py

```diff
@@ -395,7 +400,8 @@ def test_delete_volume_valid(
 
         .. test_step:: 1
 
-                Connect to Tendrl API via POST request to ``APIURL/:cluster_id/GlusterDeleteVolume``
+                Connect to Tendrl API via POST request
+                to ``APIURL/:cluster_id/GlusterDeleteVolume``
                 Where cluster_id is set to predefined value.
 
         .. test_result:: 1
```

Comment:
From where this changes are? In which commit are you making changes for this file? I tried to find it, but without success. Anyway it should have special commit.